xml/docu_styleguide.outline.xml

```diff
@@ -165,39 +157,38 @@
     <term>Preface</term>
     <listitem>
      <para>
-      Include a brief overview of the content of a
-      manual, related manuals, and typographical conventions.
-      The preface can also contain information about its target audience.
+      Include a brief overview of the content of a manual, related manuals, and
+      typographical conventions. The preface can also contain information about
+      its target audience.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry xml:id="vle-part">
     <term>Parts</term>
     <listitem>
      <para>
-      If you are writing a book with many chapters, create parts at the
-      outline level above chapters. Parts should contain at least three
-      chapters. Keep part titles clear and concise. Often a single noun is
-      enough. Typical part titles are <emphasis>Installation</emphasis>
-      or <emphasis>Network</emphasis>.
+      If you are writing a book with many chapters, create parts at the outline
+      level above chapters. Parts should contain at least three chapters. Keep
+      part titles clear and concise. Often a single noun is enough. Typical
+      part titles are <emphasis>Installation</emphasis> or
+      <emphasis>Network</emphasis>.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry xml:id="vle-chapter">
     <term>Chapters</term>
     <listitem>
      <para>
-      Chapters typically consist of the following elements (appendixes
-      should be regarded an exception):
+      Chapters typically consist of the following elements (appendixes should
+      be regarded an exception):
      </para>
      <itemizedlist>
       <listitem>
        <formalpara>
         <title>Abstract</title>
         <para>
-         Use an abstract to summarize the information provided in a book,
-         article, chapter, or set in 70&ndash;150 characters. Summarize the
-         topic instead of summarizing the outline. See also <xref
+         In an abstract, summarize the topic instead of summarizing the
+         outline. See also <xref
```

Comment:
```suggestion
         In a chapter abstract, condense the information into 70–150 characters. 
         Summarize the topic instead of summarizing the outline. See also <xref
```